src/FirmwarePlugin/APM/ArduPlaneFirmwarePlugin.cc

```diff
@@ -38,6 +38,8 @@ ArduPlaneFirmwarePlugin::ArduPlaneFirmwarePlugin(void)
     , _qAutotuneFlightMode   (tr("QuadPlane AutoTune"))
     , _qAcroFlightMode       (tr("QuadPlane Acro"))
     , _thermalFlightMode     (tr("Thermal"))
+    , _loiter2qlandFlightMod (tr("Loiter to QLand"))
```

Comment:
CI was failing again, it seems there is a typo here, "_loiter2qlandFlightMod", final "e" is missing.

Reply:
cut and paste error + my dyslexia..thanks